### PR TITLE
Guide: adding Tina to Next.js

### PR DESCRIPTION
**Pull request by ashleyjanedoiron**
Moving current [Adding Tina to Next.js](https://tina.io/guides/nextjs/adding-tina/overview/) guide to be the main 'Getting Started with TinaCMS' guide. 

This means overhauling this [current guide](https://tina.io/guides/tinacms/getting-started/introduction/). I have already merged some things from this guide into the new Next.js guide but I'm not entirely sure what is still current/relevant to document so would love another set of eyes from the OS team! 

#938

### CODE REVIEW

**Comment by DirtyF** [comment]
@ashleyjanedoiron Do you want me to follow https://tinacms-site-next-git-updated-tinacms-guide-tinacms.vercel.app/guides/tinacms/adding-tina/project-setup/ and tell you if it works?

**Comment by ashleyjanedoiron** [comment]
> @ashleyjanedoiron Do you want me to follow https://tinacms-site-next-git-updated-tinacms-guide-tinacms.vercel.app/guides/tinacms/adding-tina/project-setup/ and tell you if it works?

I walked through this guide today @DirtyF and made sure things worked. If you have the time and want to go through it on your own and let me know any suggestions, that would be great!

File: content/guides/tinacms/adding-tina/project-setup.md [hunk 1]
```diff
@@ -2,7 +2,13 @@
 title: Project Setup
 ---
 
-We're going to use the Next.js [blog starter](https://github.com/zeit/next.js/tree/canary/examples/blog-starter) as the base for our project. Create a new project by running the following commands in your terminal:
+We're going to use the Next.js [blog starter](https://github.com/zeit/next.js/tree/canary/examples/blog-starter) as the base for our project.
+
+> **The Finished Product**
+>
+> The repository at [dwalkr/next-tina-blog-starter](https://github.com/dwalkr/next-tina-blog-starter) contains the end product of this guide. Feel free to fork it and follow along!
```
Comment (DirtyF) [inline content/guides/tinacms/adding-tina/project-setup.md:9]: Why don't we host this on our org?

**Comment by DirtyF** [comment]
Working as expected, noticed you can't just copy and paste on the forms setup.

Maybe we can add a preview of the expected result in this guide?

CleanShot 2021-07-26 at 14 38 13@2x

**Review by DirtyF — APPROVED** [review]
LGTM

**Comment by perkinsjr** [comment]
This can be merged today @ashleyjanedoiron as we aren't dependant on anything else.

**Pull request merged**